app/views/templates/_news_and_documents.html.erb

```diff
@@ -45,34 +49,4 @@
     </div>
   </div>
 
-  <div class="row column">
-    <%
-      #article_types = get_article_types
-
-      #article_types.each do |type|
-       # concat "<p>#{type.name}</p>".html_safe
-      #end
-
-    %>
-
-    <div>
-    
-      
-    </div>
-
-  </div>
-
-  <div class="row column">
-  <% #@cms_page.children.published.each do |p| %>
-    
-    <div>
-      <h3><%#= link_to p.label, p.url(:relative) %></h3>
-      <%#= image_tag cms_block_content(:image, p).file.url(:small), alt: p.label %>
-      <div>
-      <%#= cms_block_content(:introduction, p) %>
-      </div>
-    </div>
-  <% #end %>
-  </div>
-
```

Comment:
This has all been deleted now though, so not sure there is much I can do to make it not appear in the PR.